serial_communication/web/app.py

```diff
@@ -24,5 +67,41 @@ def send_serial():
     return {'status': 'success'}
 
 
+def fetch_current_time_online():
```

Comment:
Don't use fetch_time....py code here, do it in separate file just call it form this code.